y/y.go

```diff
@@ -179,6 +183,30 @@ func FixedDuration(d time.Duration) string {
 	return str
 }
 
+// BuildChecksum builds sha256 checksum for the given data.
+func BuildChecksum(data []byte) ([]byte, error) {
+	sum := sha256.New()
+	bytesWritten, err := sum.Write(data)
+	if bytesWritten != len(data) {
+		return nil, errors.New("Size mismatch between bytes written and length of data")
```

Comment:
May be, declare this just once

Reply:
We might not reuse this error. It's too specific. We can keep it here.